### ISSUE TITLE
Specify how Opus's duplicate pre-skip info should be handled 

### ISSUE THREAD

**Issue by felicialim** [AOMediaCodec/iamf#250]
The same info is defined in two places: 

1. `roll_distance` in `codec_config()`
2. Pre-skip field of the  RFC7845 ID Header (which is used for `decoder_config('opus')`)

One option is to require that both are identical.

**Comment by sunghee-hwang** [comment]
Section 4.6 of RFC7845 is coped in below.
```
4.6. Seeking and Pre-roll
 Seeking in Ogg files is best performed using a bisection search for a
 page whose granule position corresponds to a PCM position at or
 before the seek target. With appropriately weighted bisection,
 accurate seeking can be performed in just one or two bisections on
 average, even in multi-gigabyte files. See [SEEKING] for an example
 of general implementation guidance.
 When seeking within an Ogg Opus stream, an implementation SHOULD
 start decoding (and discarding the output) at least 3840 samples
 (80 ms) prior to the seek target in order to ensure that the output
 audio is correct by the time it reaches the seek target. This
 "pre-roll" is separate from, and unrelated to, the pre-skip used at
 the beginning of the stream. If the point 80 ms prior to the seek
target comes before the initial PCM sample position, an
 implementation SHOULD start decoding from the beginning of the
 stream, applying pre-skip as normal, regardless of whether the pre-
 skip is larger or smaller than 80 ms, and then continue to discard
 samples to reach the seek target (if any).
```
I think that it does not need to be the same.

**Comment by tdaede** [comment]
Double check that these are both pre-skip (discarded samples) and not pre-roll (discarded samples after seeking).

**Comment by tdaede** [comment]
`roll_distance` is for seeking, but `num_samples_to_trim_at_start` is duplicative of the preskip field in rfc7845.

**Comment by sunghee-hwang** [comment]
it can be closed because #288 will address this as well.

**Comment by tdaede** [comment]
This is now handled.
